apps-cd/requirements.txt

```diff
@@ -9,6 +9,7 @@ google-auth==1.6.3
 google-auth-httplib2==0.0.3
 google-cloud-core==1.0.3
 google-cloud-storage==1.17.0
+jinja2==2.11.2
```

Comment:
Do you know, which version of python we are using to run release infra scripts?

Reply:
https://github.com/kubeflow/testing/blob/e72b7f567195d3acc657b02e863e73c403d7e595/apps-cd/Dockerfile#L8